vuu-ui/packages/vuu-layout/src/flexbox/Flexbox.tsx

```diff
@@ -10,12 +9,10 @@ const classBase = "hwFlexbox";
 
 const Flexbox = forwardRef(function Flexbox(
   props: FlexboxProps,
-  ref: ForwardedRef<HTMLDivElement>
```

Comment:
the forwardedRef here isn't strictly needed - it's a courtesy for users who might want to capture a ref to the root element. If we DID want to remove the forwardedRef, though, the forwardRef call against the component should be removed as well.